### PR TITLE
fix(component prefilling): added to design v2

### PR DESCRIPTION
**Pull request by JSchenk8**
Updated the component prefilling docs with example config and component property

### CODE REVIEW

File: content/reference-docs/document/document-design/_index.md [hunk 1]
```diff
@@ -517,3 +517,47 @@ fieldExtractor: [
...
 - `matches` defines an array of component.directive pairs from which the content should be synced into the metadata field. E.g. for a component named "title" that has a doc-editable directive "title", you would write "title.title". When `type` is set to `cssProperty` one can define which CSS property to extract, e.g. `color`.
+
+### Component Prefilling
+
+Livingdocs supports two types of component prefilling; author and author-email.
+
+Adding a component prefilling array to your `design_settings.js` file will enable the feature:
+
+```js
+prefilledComponents: [{
```
Comment (DaRaFF) [inline content/reference-docs/document/document-design/_index.md:528]: I reviewed some document PRs recently with configs and I think it's much clearer if you show the whole Project Config object (you could fix that in the whole file).

<<<<
You can enable Prefilled Components by adding them to the Design Settings.

```js
{
  v: 2,
  ...
  designSettings: {
    prefilledComponents: [{
      component: 'title',
      content: [{
        directive: 'author',
        value: 'author', // can be 'author' or 'author-date'
        type: 'computed'
      }]
    }]
  }
}
```
<<<<

File: content/reference-docs/document/document-design/_index.md [hunk 2]
```diff
@@ -517,3 +517,47 @@ fieldExtractor: [
...
+    component: 'title',
+    content: [{
+      directive: 'author',
+      value: 'author', // can be 'author' or 'author-date'
+      type: 'computed'
+    }]
+  }]
+```
+
+Make sure you then have a directive in your component which can be prefilled, for example:
```
Comment (DaRaFF) [inline content/reference-docs/document/document-design/_index.md:538]:
```suggestion
As a second step, you have to add a Design Component `title` which contains a directive `author`. After that when you add the `title` component to a document, the `author` should be prefilled automatically.
```

**Review by DaRaFF — CHANGES_REQUESTED** [review]
Thanks for adding that! Please fix my feedback. We still can work a lot on making descriptions/configs more clear and I still try to find out the best way to do that.

**Comment by JSchenk8** [comment]
@DaRaFF thanks very much for the thorough review, makes a lot of sense! I will take this into other documentation edits too, let me know how this version is now 🚀 🙏

**Review by DaRaFF — APPROVED** [review]
Yep, now it looks good, thx 👍

**Pull request merged**